### PR TITLE
Add Show Comments button

### PR DESCRIPTION
**Pull request by danroth27**
@shirhatti Please review

### CODE REVIEW

**Comment by danroth27** [comment]
@javiercn @Rick-Anderson @rachelappel @Erikre Please review

**Comment by javiercn** [comment]
:shipit:

**Comment by shirhatti** [comment]
I propose you do one of these two things:

1. Replace the Show comments with Show/Hide toggle functionality
2. Hide/Disable the Show comments button after you press it.

Right you can repeatedly press the show comments, and there is no feedback to indicate nothing happens after the first time you press it

**Comment by danroth27** [comment]
@shirhatti I opted for #2.

**Comment by Rick-Anderson** [comment]
#2 is the right choice.

File: common/_static/disqus.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  if (button.length)
+  {
+    var handler = function(){
+      /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
+      var disqus_shortname = 'aspnetdocs'; // required: replace example with your forum shortname
+
+      /* * * DON'T EDIT BELOW THIS LINE * * */
+      var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
+      dsq.src = 'https://' + disqus_shortname + '.disqus.com/embed.js';
+      (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
```
Comment (shirhatti) [inline common/_static/disqus.js]: You've used a JQuery style DOM selector in line 1 and a vanilla js DOM selector in line 13. Maybe try and be consistent?

**Comment by shirhatti** [comment]
:shipit:

File: common/_static/disqus.js (hunk 1 above)
Comment (danroth27) [inline common/_static/disqus.js]: This part of the code is the recommended code snippet from the disqus docs, so I'm leaving it untouched.

**Comment by danroth27** [comment]
cd2b93ccd18fe57cbdf7f71e61349fdc4a26ce3f

**Pull request closed without merging**